docs/Converters/GENFItoBIDS.md

```diff
@@ -17,6 +17,14 @@ Please note that this converter currently processes the following modalities :
 - Fieldmaps
 - rsfMRI
 
+## Downloading GENFI
+
+To download GENFI in a way that you can convert it, you only need to make sure of two thing: 
+- To only select the "simplify downloaded archive structure" in download data options.
+- And in Select Image Data section Scans Format, select only "DICOM"
```

Comment:
I'm not sure my suggestion is correct, please verify where the `"` should be placed.

```suggestion
- Select only "DICOM" in "Select Image Data" section "Scans Format".
```